identity-map/README.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: pattern
+title: Identity Map
+folder: identity-map
+permalink: /patterns/identity-map/
+categories: Data access
+language: en
+tags:
+- Performance
+---
```

Comment:
Please check the YAML frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute. Basically we can leave out some of these tags.